sos/report/plugins/insights.py

```diff
@@ -52,8 +52,8 @@ def setup(self):
             timeout=30
         )
 
-        for _dir in ["/etc/rhsm", "/sys/kernel", "/var/lib/sss"]:
-            self.add_cmd_output(f"/bin/ls -lanR {_dir}", cmd_as_tag=True)
+        self.add_dir_listing(["/etc/rhsm", "/sys/kernel", "/var/lib/sss"],
+                             recursive=True)
```

Comment:
We stop adding tags here. Also not sure if `-n` is important or not (`like -l, but list numeric user and group IDs`).

I am checking this internally..